spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -444,7 +444,7 @@ def main(argv=None):
     # Normalize CSA values (MEAN(area))
     if arguments.normalize is not None:
         data_subject = pd.DataFrame([arguments.normalize])
-        path_model = os.path.join(__sct_dir__, 'spinalcordtoolbox', 'data', 'csa_normalization_models',
+        path_model = os.path.join(__sct_dir__, 'data', 'csa_normalization_models',
                                   '_'.join(sorted(data_subject.columns)) + '.csv')
```

Comment:
This path is referring to some actual files in the SCT repository: [`spinalcordtoolbox/data/csa_normalization_models`](https://github.com/jqmcginnis/spinalcordtoolbox/tree/jqm/3558-csa-normalization-model-path/spinalcordtoolbox/data/csa_normalization_models)

So, if the path in the code changes, then the location of the files has to change too.

Could you please take the folder `data/` and move it up one level? In other words:

* Before: `./spinalcordtoolbox/data/csa_normalization_models/`
* After: `./data/csa_normalization_models/`

Reply:
Hi, thank you very much for reviewing and discovering this! Just updated it in this commit c.f. [8e959b7](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3821/commits/8e959b7dd2f8b86b325ca551e939fa708624a479) :slightly_smiling_face: